content/developer/howtos/website_themes/animations.rst

```diff
@@ -0,0 +1,44 @@
+==========
+Animations
+==========
+
+Eye-catching animations can bring your website to life.
+
+On appearance
+=============
+
+In standard, you can add animations to columns when they appear, thanks to the Website Builder. Odoo
+detects when your element is in the viewport and launches the animation. A large selection of
+animations is available:
+
+- Fade in
+- Bounce in
+- Rotate in
+- Zoom in
+- …
+
+You can easily define an animation on a column in your custom theme. You need to add two classes:
+`o_animate` and `o_anim_fade_in`. The second class changes depending on the type of animation you
+want to use.
+
+Add the `o_animate_both_scroll` class to launch the animation everytime the column appears on the
+screen. The animation is only launched once by default.
+
+You can also define an `animation-duration` and an `animation-delay` directly in the `style`
+attribute.
+
+**Use**
+
+.. code-block:: xml
+
+   <div class="col-lg-6 o_animate o_anim_fade_in o_animate_both_scroll" style="animation-duration: 2s !important; animation-delay: 1s !important;">
+       <h2>A Section Subtitle</h2>
+       <p>Write one or two paragraphs describing your product or services.</p>
+   </div>
+
+.. image:: animations/animations.png
+   :alt: Animation options
+
+.. seealso::
+   `Website Animate
+   <{GITHUB_PATH}/addons/website/static/src/scss/website.scss#L1605>`_
```

Comment:
What I meant by "freeze" the link is "include a specific commit in the URL" so that the referenced line doesn't move.
For example https://github.com/odoo/odoo/blob/fb87640322b0954aa91e6ba4e30c0d78d9d0f7b3/addons/website/static/src/scss/website.scss#L1642

Reply:
@AntoineVDV I discussed this with NMI, we thought it would be better to add a link to the file without mentioning a specific line. This way the code is always up to date and not linked to a specific commit. However we can change it if necessary :)